static/app/bootstrap/exportGlobals.tsx

```diff
@@ -38,8 +38,10 @@ const SentryApp = {
   },
 
   // The following components are used in legacy django HTML views
+  // or in the Sentry sandbox
   ConfigStore: require('sentry/stores/configStore').default,
   HookStore: require('sentry/stores/hookStore').default,
+  GuideActionCreator: require('sentry/actionCreators/guides'),
```

Comment:
damn

Reply:
@evanpurkhiser That's how the sandbox can call it 🤷

Reply:
Why can't we detect demo mode?

Reply:
@evanpurkhiser We can, but then how do we call the `setForceHide(false)` after the email was added?

Reply:
Ah damn, yeah I see :(

Reply:
@evanpurkhiser If we generated a single JS file with the sandbox + sentry code we could avoid it, but that's a lot of work. Maybe one day we could do it but for now we're doing fine with compiling the sandbox code separately and exposing the interface on `window`.